pandas/formats/format.py

```diff
@@ -2260,6 +2260,35 @@ def _format_strings(self):
         return fmt_values
 
 
+def format_percentiles(percentiles):
+    '''
+    Outputs rounded and formatted percentiles. Rounding preserves uniqeness.
+
+    Parameters
+    ----------
+    percentiles - 1d ndarray of strictly increasing floats from interval [0,1]
```

Comment:
list-like, you should sort them in here, don't suppose that they are passed in correctly.